src/_fastmath.c

```diff
@@ -1342,6 +1353,10 @@ rabinMillerTest (mpz_t n, int rounds, PyObject *randfunc)
 	}
 
 cleanup:
+	for (i = 0; last_tested_cleanup >= 0 && i <= last_tested_cleanup; ++i)
```

Comment:
You set `i = 0`, so you don't need the `last_tested_cleanup >= 0 &&`, but this works.